src/main/java/com/google/sps/data/GraphDataObject.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GraphDataObject {
+  List<Object> hourList = new ArrayList<>();
+  List<Object> difficultyList = new ArrayList<>();
+
+  public GraphDataObject(String hourTitle, String diffTitle) {
+    List<String> titleContainer = new ArrayList();
+    titleContainer.add(hourTitle);
+    hourList.add(titleContainer);
+
+    titleContainer.removeAll(titleContainer);
```

Comment:
What does this do?

Reply:
The Charts API says I need to have a name for the data so I need to have it as the first element of the data array. Also the format has to be [[title],[data1],[data2],....] that is why I am wrapping each data point in an array.